src/csvcubed/readers/cubeconfig/v1/columnschema.py

```diff
@@ -90,7 +90,7 @@ def map_to_new_qb_dimension(
             # If so, then the code_list provided is not a boolean value or looks like an uri.
             if not isinstance(self.code_list, bool):
                 if looks_like_uri(str(self.code_list)) == False:
-                    raise BaseException(
+                    raise UserWarning(
```

Comment:
Sorry if I misunderstood this, do we want this condition to terminate the program? Or simply give a warning and continue? If a warning is fine then this looks ready.